app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,28 @@
+class RelationshipsController < ApplicationController
+  before_action :require_user, only: [:create, :destroy, :index]
+
+  def index
+    @followings = current_user.followings
+  end
+
+  def create
+    @relationship = current_user.relationships.create(following_id: params[:following_id])
+
+    if @relationship.errors.any?
+      @relationship.errors.messages.values.each do |msg|
```

Comment:
You could use this `@relationship.errors.full_messages.each do |error|` to return just the array of errors :).